arches_lingo/src/arches_lingo/components/sidenav/SideNav.vue

```diff
@@ -41,13 +41,13 @@ const items = ref([
     display: flex;
     flex-direction: column;
     align-items: center;
-    min-width: 3rem;
+    min-width: var(--p-button-icon-only-width);
```

Comment:
just for my own understanding, where is this declared?

Reply:
PrimeVue

Reply:
https://primevue.org/button/#theming.tokens